pipeline/inputs/kubernetes-events.md

```diff
@@ -50,4 +50,4 @@ In the following configuration file, the input plugin *kubernetes_events* collec
 ```
 
 ### Event Timestamp
-Event timestamp will be created from the first existing field in the following order of precendence: lastTimestamp, firstTimestamp, metadata.creationTimestamp
+Event timestamp will be created from the first existing field in the following order of precedence: lastTimestamp, firstTimestamp, metadata.creationTimestamp
```

Comment:
Added line wrapping, removed passive voice, and turned the list into an ordered list to better indicate the order of precedence.

```suggestion

Event timestamps are created from the first existing field, based on the following
order of precedence:

1. `lastTimestamp`
1. `firstTimestamp`
1. `metadata.creationTimestamp`
```

Reply:
As a note, Markdown will turn the "1"s into "1, 2, 3".

Reply:
Thanks!